keyword-guidelines.md

```diff
@@ -0,0 +1,102 @@
+# Keyword Guidelines for the Specifications
+
+The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" as used in the specification
+documents are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt) (Bradner, S., "Key words for use in RFCs to Indicate Requirement Levels", BCP 14, RFC 2119, March 1997.)
+      
+## Introduction
```

Comment:
Which means we probably don't need an entire file for it, just add it to the top of each stand-alone doc.